python/idsse_common/idsse/common/aws_utils.py

```diff
@@ -65,19 +65,31 @@ def aws_ls(self, path: str, prepend_path: bool = True) -> Sequence[str]:
             return [os.path.join(path, filename.split(' ')[-1]) for filename in commands_result]
         return [filename.split(' ')[-1] for filename in commands_result]
 
-    def aws_cp(self, path: str, dest: str) -> bool:
-        """Execute an 'cp' on the AWS s3 bucket specified by path, dest
+    def aws_cp(self,
+               path: str,
+               dest: str,
+               concurrency: int  = 5,
```

Comment:
I'd rather the default be None, and if concurrency or chuck_size was None it wouldn't be included in the command (thus use the s5cmd default) vs us having a copy of the default values, which s5cmd could change.

Reply:
Good point. Fixed!